tests/vegalite/v5/tests/test_params.py

```diff
@@ -0,0 +1,167 @@
+"""Tests for variable parameters and selection parameters"""
+
+import pandas as pd
+
+import warnings
+import pytest
+
+import re
+
+import altair.vegalite.v5 as alt
+
+
+def test_variable_param():
+    data = pd.DataFrame([{"a": "A", "b": 28}])
+
+    rad_slider = alt.binding_range(min=0, max=20, step=1)
+    rad_var = alt.param(bind=rad_slider, value=0, name="paramName")
+
+    c = (
+        alt.Chart(data)
+        .mark_bar(cornerRadius=rad_var)
+        .encode(
+            x="a:N",
+            y="b:Q",
+        )
+        .add_params(rad_var)
+    )
+
+    dct = c.to_dict()
+
+    mark_dict = {"type": "bar", "cornerRadius": {"expr": "paramName"}}
+    param_list = [
+        {
+            "name": "paramName",
+            "bind": {"input": "range", "max": 20, "min": 0, "step": 1},
+            "value": 0,
+        }
+    ]
+
+    assert dct["mark"] == mark_dict
+    assert dct["params"] == param_list
+
+
+def test_param_expr():
+    height_var = alt.param(name="height")
+    inner_var = height_var / 2
+    tick_var = (3 * height_var) / 4
+
+    inner_str = inner_var._to_expr()
+    inner_ans = "(height / 2)"
+
+    tick_str = tick_var._to_expr()
+    tick_ans = "((3 * height) / 4)"
+
+    assert height_var._to_expr() == "height"
+    assert inner_str == inner_ans
+    assert tick_str == tick_ans
```

Comment:
```suggestion
    height_str = height_var._to_expr()
    inner_str = inner_var._to_expr()
    tick_str = tick_var._to_expr()

    assert height_str == "height"
    assert inner_str == "(height / 2)"
    assert tick_str == "((3 * height) / 4)"
```
Looks great! Improved readability a little bit here. To make it directly clear what is expected in the assertion. Furthermore all fine.

Reply:
Thank you!